docs/SECURITY.md

```diff
@@ -0,0 +1,27 @@
+# Security Policy
+
+Hello and thank you for your interest in the `urfave/cli` security
+policy! :tada: :lock:
+
+## Supported Versions
+
+| Version      | Supported                             |
+| ------------ | ------------------------------------- |
+| `>= v2.3.x`  | :white_check_mark:                    |
+| `< v2.3`     | :x:                                   |
+| `>= v1.22.x` | :white_check_mark: :lady_beetle: [^1] |
+| `< v1.22`    | :x:                                   |
+
+## Reporting a Vulnerability
+
+Please disclose any vulnerabilities by sending an email to:
+
+[dan+urfave-cli-security@meatballhat.com](mailto:dan+urfave-cli-security@meatballhat.com)
```

Comment:
There is no team/maintainer email group address this can be sent to ?

Reply:
Not currently, but I would really like to have one! Do you know of a reputable provider for group email (excluding Google Groups, which is starting to smell a lot like a discontinued product)?

Reply:
update: librelist is _not_ still an option